test/chplcheck/PREDIFF

```diff
@@ -8,14 +8,21 @@ FLAGS="--enable-rule ConsecutiveDecls"\
 " --internal-prefix myprefix_ --internal-prefix _"\
 " --skip-unstable"
 
+# read extra arguments from a $1.chplcheckopts file
+# currently only supports 1 line with nothing but flags
+if [ -e $1.chplcheckopts ]; then
+  FLAGS="$FLAGS $(cat $1.chplcheckopts)"
+fi
+
+
 $CHPL_HOME/tools/chplcheck/chplcheck \
-  $FLAGS $1.chpl >> $2
+  $FLAGS $1.chpl >>$2 2>>$2
```

Comment:
does &>>$2 work here?

Reply:
It should

Reply:
Huh, well rather than working as expected (and as the [bash manual](https://www.gnu.org/software/bash/manual/html_node/Redirections.html) claims it should), `&>>$2` does not work.

On my Mac with bash it doesn't work, with zsh it does. Weird.

But for the purposes of this PR, I am going to use `>>$2 2>&1`, which works the same and does work.